web/viewer.css

```diff
@@ -920,6 +920,24 @@ html[dir="rtl"] #viewOutline.toolbarButton::before {
   content: url(images/toolbarButton-search.png);
 }
 
+.toolbarButton.pdfSidebarNotification::after {
+  position: absolute;
+  display: inline-block;
+  top: 0px;
+  /* Create a filled circle, with a diameter of 10 pixels, using only CSS: */
+  content: '';
+  background-color: #8BE651;
+  height: 10px;
```

Comment:
You're right, I mixed that up. Personally I feel that the tooltip there was only useful because it was about an instruction that could not be displayed visually. Here, however, we have the visual notification. In my opnion, that should be enough, but I'm not totally against adding a tooltip, so if you think it would be more clear to add the tooltip as well, I'm fine with that.